vsintegration/src/FSharp.Editor/Completion/CompletionProvider.fs

```diff
@@ -45,7 +45,7 @@ type internal FSharpCompletionProvider
         |> List.filter (fun (keyword, _) -> not (PrettyNaming.IsOperatorName keyword))
         |> List.sortBy (fun (keyword, _) -> keyword)
         |> List.mapi (fun n (keyword, description) ->
-             CommonCompletionItem.Create(keyword, Nullable Glyph.Keyword, sortText = sprintf "%06d" (1000000 + n))
+             CommonCompletionItem.Create(keyword, CompletionItemRuls.Default, Nullable Glyph.Keyword, sortText = sprintf "%06d" (1000000 + n))
```

Comment:
This is a port from your last PR (last week?) where I had to make this change.  Is this change not in your latest PR dotnet/roslyn#19525?